npm/src/index.ts

```diff
@@ -1 +1,28 @@
-// This is the entrypoint file.
+import { GraphqlClient } from "./graphql"
+
+// All queries can be imported from this package. You add them in the same way you do as for the scala classes.
+import {
+  GetSeries,
+  GetSeriesQueryVariables,
+  GetSeriesQuery
+} from "@nationalarchives/tdr-generated-graphql"
+import { ApolloQueryResult } from "apollo-boost"
+
+// Needed to keep typescript happy
+declare var TDR_API_URL: string
+
+// Pass this class instance to any class or function that needs it
+// This means we don't create the graphql client each time we need a query or mutation
+const graphqlClient = new GraphqlClient(TDR_API_URL)
+
+// Then use the imported types like this
```

Comment:
Is this just here as an example? If so maybe make an explicit comment that this is the case

Reply:
Yeah it is, I put it in the commit comment but it makes sense to put it in here as well. I've added it in.